lms/templates/ccx/grade_book_download.underscore

```diff
@@ -0,0 +1,26 @@
+<p>
+  <a href="#" id="ccx-calculate-grades-csv"><%- gettext('Generate Grade Report') %></a>
+</p>
+<br/>
+
+<% if (showSuccessMessage) { %>
+<div class="request-response msg msg-confirm copy" id="ccx-gradebook-request-response"></div>
+<% }  %>
+
+<% if (showErrorMessage) { %>
+<div class="request-response-error msg msg-error copy" id="ccx-gradebook-request-response-error"></div>
+<% }  %>
+<br/>
+
+<p><b> <%- gettext('Grade Book Available for Download') %></b></p>
+<%
+  // Translators: a table of URL links to report files appears after this sentence.
```

Comment:
I don't think this translators comment will be picked up, since there is a `<p>` between the comment and the string being translated.